src/addons/dragAndDrop/WeekWrapper.js

```diff
@@ -61,8 +61,12 @@ class WeekWrapper extends React.Component {
   }
 
   update(event, start, end) {
+    const { accessors } = this.props
+    const updatedEvent = accessors.eventUpdater(event, { start, end })
```

Comment:
ideally this shouldn't be an accessor, since it's not accessing anything. it makes more sense as a separate prop. It's also not updating the event it's creating a new pseudo event